client/components/OnboardingWizard.jsx

```diff
@@ -12,22 +13,48 @@ import StepDiet from "./onboarding/StepDiet";
 import StepMetrics from "./onboarding/StepMetrics";
 import StepPhone from "./onboarding/StepPhone";
 import ProgressBar from "./onboarding/ProgressBar";
+import "./onboarding/onbooarding-styles/OnboardingWizard.css";
 
 export default function OnboardingWizard() {
   const [currentStep, setCurrentStep] = useState(0);
   const [formData, setFormData] = useState({
-    name: "",
+    name: { firstName: "", lastName: "", },
     age: "",
-    goal: "",
+    goal: [],
     gender: "",
     occupation: "",
     availability: "",
     preference: "",
     diet: "",
-    metrics: "",
+    metrics: { height: "", weight: "" },
     phone: "",
   });
 
+  useEffect(() => {
+    const handleTooltipClick = (e) => {
+      if (e.target.classList.contains("tooltip")) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        document.querySelectorAll(".tooltip").forEach((tooltip) => {
+          tooltip.classList.remove("active");
+        });
+
+        e.target.classList.add("active");
+      } else {
+        document.querySelectorAll(".tooltip").forEach((tooltip) => {
+          tooltip.classList.remove("active");
+        });
+      }
```

Comment:
`document.querySelectorAll(".tooltip")` is called twice in both if-else. This can be refactored to avoid `else` clause.
```suggestion
      const tooltip = e.target.closest(".tooltip");

      const tooltips = document.querySelectorAll(".tooltip");
      tooltips.forEach((el) => el.classList.remove("active"));
    
      if (tooltip) {
        e.preventDefault();
        e.stopPropagation();
        tooltip.classList.add("active");
      }
```

Reply:
Thanks for catching that!